src/main/resources/assets/tconstruct/book/materials_and_you/en_us/tools/axe.json

```diff
@@ -2,19 +2,19 @@
   "tool": "tconstruct:axe",
   "text": [
     {
-      "text": "The Hatchet is a basic chopping tool. It is effective on wood and leaves. Breaking leaves with it does not consume durability."
+      "text": "The Hand Axe is a basic chopping tool. It is effective on wood and leaves."
     },
     {
       "text": "\n"
     },
     {
-      "text": "Using the hatchet in combat allows you to disable the shield of your opponent."
+      "text": "It is like a vanilla axe; a weapon or a tool, or both. The choice is yours."
```

Comment:
"Vanilla axe" is a bit too meta. Just say it works as both a weapon and a tool.

Shield bash is also still relevant IIRC, if its not working thats a bug, so I guess that could be fixed later.